src/components/common/Header.tsx

```diff
@@ -20,7 +19,7 @@ export const Header = React.memo(() => {
         </Heading>
       </Button>
       {user.profileImage != null && (
-        <Avatar url={`url(${user.profileImage.url})`} />
+        <Avatar url={`url(${user.profileImage?.url ? user.profileImage.url : '/empty-avatar.png'})`} />
```

Comment:
Please use an icon here instead of the png, also this could be `user.profileImage?.url ?? '/empty-avatar.png'`

Reply:
How do you put an Icon inside a Avatar though?

Reply:
Might need a new component